CodingChallenge.PirateSpeak/Solution.cs

```diff
@@ -8,37 +8,43 @@ public class Solution
     {
         public string[] GetPossibleWords(string jumble, string[] dictionary)
         {
+            // Solution using Linq
+
+            return dictionary.Where(x => String.Join("", x.OrderBy(c => c)).ToLower() == String.Join("", jumble.OrderBy(j => j)).ToLower()).ToArray();
```

Comment:
very nice.  only thing here I'll leave you with is that the `String.Join("", jumble.OrderBy(j => j)).ToLower()` never changes, but we still evaluate it for each jumble.  could pull that out into a variable and only do that operation once.  Its a micro-optimization in this case, just something to think about.

Reply:
Ah very good point. I was too focused on making it all happen in one line and didn't think about that implication.